### PR TITLE
Update Place Tomba Foulah

### PR DESCRIPTION
**Pull request by 5212BABA**
Updating `data/125/952/004/3/1259520043.geojson` using the [WOF Editor](https://github.com/iandees/wof-editor)

### CODE REVIEW

**Comment by stepps00** [comment]
@5212BABA there are several pull requests in this repo, and they'll all need adjustments before anything can be merged. Is it possible to list out all of the changes you're requesting in this repo? I imagine few records need updates, but a few new records need to be created. If you're able to list out what places you expect to see, their lat/lng location, and any source information (like GeoNames), we can take a look.

**Comment by 5212BABA** [comment]
Ok I understood I really wanted to contribute to github but I'm a beginner here I don't even know how to list places?  I can only make changes to the places whose geographical names and lat/lon coordinates I know

File: data/125/952/004/3/1259520043.geojson [hunk 1]
```diff
@@ -53,8 +53,8 @@
         }
     ],
     "wof:id":1259520043,
-    "wof:lastmodified":1566639834,
-    "wof:name":"Tomba Foulah",
+    "wof:lastmodified":1684699670,
+    "wof:name":"Tomba dougoula ",
```
Comment (nvkelso) [inline data/125/952/004/3/1259520043.geojson:57]: Watch for trailing space characters (they should be removed)

File: data/125/952/004/3/1259520043.geojson [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "id": 1259520043,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
+    "edtf:cessation":"uuuu ",
```
Comment (nvkelso) [inline data/125/952/004/3/1259520043.geojson:5]: Watch for trailing space characters (they should be removed)

**Comment by nvkelso** [comment]
Replaced with https://github.com/whosonfirst-data/whosonfirst-data-admin-gn/pull/37

**Pull request closed without merging**